### PR TITLE
Adding ejrober's coordinates

### LINKED ISSUE #9710: Please, add me
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by ejrober**
@githubteacher What do you think?

This pull request closes #9710.

### CODE REVIEW

**Comment by seanbigler** [comment]
@ejrober Looks good.

File: _pins/ejrober.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: YOUR-USERNAME
```
Comment (hectorsector) [inline _pins/ejrober.json]: @ejrober you'll need to include your username here

**Comment by GarrettRVD** [comment]
Awesome!

**Comment by Coltyboy11** [comment]
Looks great!

**Comment by YuliyaKar** [comment]
@ejrober Great job! ;)

**Comment by hectorsector** [comment]
@billyjoebaron we appreciate your willingness to help, but please allow the author of issues and PRs to close or merge their own.

**Comment by hectorsector** [comment]
@ejrober your code looks perfect! Please go ahead and click the big green button below to **Merge pull request**. In an effort to keep things in this repository clean, I will merge this if it is still open on Friday.

**Comment by gooshtie** [comment]
NICE!!!!

**Pull request merged**